docs/UserGuide.md

```diff
@@ -54,7 +54,7 @@ For a full list of commands and detailed instructions, head to the [Features](#f
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
-2. Download the latest `teacherpet.jar` from https://github.com/AY2223S1-CS2103T-T09-4/tp/releases when made available.
+2. Download the latest `teacherspet.jar` from https://github.com/AY2223S1-CS2103T-T09-4/tp/releases when made available.
```

Comment:
I believe this is named `teachersPet.jar` on GitHub right?

Reply:
Excellent catch Xinyu! Thanks for the input 👍